src/App.tsx

```diff
@@ -66,6 +68,24 @@ const App: React.FC = () => {
     }
   }, [setAppInfo, setUserInfo, client]);
 
+  const executeWfsDescribeFeatureType = useExecuteWfsDescribeFeatureType();
+  const getPropertyNames = useCallback(async (layerId: number | undefined) => {
+    let response: DescribeFeatureType | undefined;
+    const propertyNames: string[] = [];
+    if (layerSuggestionList && layerId) {
+      const layer = layerSuggestionList.filter(item => item.id === layerId)[0];
+      if (layer) {
+        response = await executeWfsDescribeFeatureType(layer);
```

Comment:
What happens if `executeWfsDescribeFeatureType` throws an error? Can we show the error (or at least the missing option to get the list of available properties) to the user somehow?

Reply:
resolved in new commit